src/theme/components/spinner.ts

```diff
@@ -0,0 +1,9 @@
+export const Spinner = {
+  baseStyle: {
+    thickness: "4px",
+    speed: "0.65s",
+    emptyColor: "gray.200",
+    color: "brand.teal.400",
+    size: "xl",
+  },
+}
```

Comment:
This is the preferred method to customize a Chakra component, rather than importing and re-exporting with props.